### PR TITLE
README.md add gradle support

### PR DESCRIPTION
**Pull request by GeorgeYuen**

### CODE REVIEW

**Comment by GeorgeYuen** [comment]
QAQ why  CI build failed?

File: README.md [hunk 1]
```diff
@@ -61,6 +61,21 @@ version can be found on in the maven repository for
 </dependency>
 ```
 
+If you use Gradle,you can add the dependency.
```
Comment (checketts) [inline README.md]: Add space after comma. rephrase `you can add the dependency`

**Comment by checketts** [comment]
The test failures were https://travis-ci.org/prometheus/client_java/builds/238184202#L7629

Looks like a timing issue that should be improved 

```
MethodTimerTest.timeMethod:43 expected:<0.02> but was:<0.03920238>
MethodTimerTest.testValueParam:71 expected:<0.035> but was:<0.04848616>
```

**Comment by brian-brazil** [comment]
There's a variety of build systems out there, I'm not maintaining the version numbers for all of them. As maven is the closest thing to a standard, that's the one we list.

**Pull request closed without merging**